### DISCUSSION TITLE
官网的live demo  不能运行

### DISCUSSION THREAD

**Discussion by ahuigo** [antvis/X6#3496] [Show and tell]
官网的demo  有点问题（stackbitz 、codeSandBox）

https://x6.antv.antgroup.com/examples/showcase/practices/#flowchart

https://stackblitz.com/run?file=index.ts
https://codesandbox.io/s/swso1x

**Comment by liaodalin19903** [comment]
具体报错信息贴一下。

**Reply by ahuigo** [reply]
https://x6.antv.antgroup.com/examples/showcase/practices/#flowchart
https://stackblitz.com/run?file=index.ts
https://codesandbox.io/s/swso1x

![image](https://user-images.githubusercontent.com/1781999/231678724-a074f51e-4972-44c5-9136-6a34b61c27f9.png)
![image](https://user-images.githubusercontent.com/1781999/231678921-906639b3-fe15-475e-b754-d33d4ce2b164.png)

**Reply by ahuigo** [reply]
@liaodalin19903 文档图例的repo 在哪里改呢？想加一下正确的showcase link:
https://codesandbox.io/s/kind-hodgkin-3ynmos?file=/index.ts

**Reply by liaodalin19903** [reply]
你好，这个报错信息是dnd 这个依赖没有找到，你看你的依赖中没有：
![image](https://user-images.githubusercontent.com/12727582/233069383-669bf456-16a6-4806-b945-5f6506e66f6f.png)

**Reply by ahuigo** [reply]
> 你好，这个报错信息是dnd 这个依赖没有找到，你看你的依赖中没有： ![image](https://user-images.githubusercontent.com/12727582/233069383-669bf456-16a6-4806-b945-5f6506e66f6f.png)

这我知道，另外css 也有问题。所以请在文档中更新一下正确的showcase吧（我不知道这个文档在哪里改，不能提PR）：
https://codesandbox.io/s/kind-hodgkin-3ynmos?file=/index.ts
